validations/src/validations.rs

```diff
@@ -215,55 +220,74 @@ fn update_liars(liars: &mut Vec<bool>, item: RadonTypes, condition: bool) -> Opt
     }
 }
 
+/// An `Either`-like structure that covers the two possible return types of the
+/// `evaluate_tally_precondition_clause` method.
+#[derive(Debug)]
+pub enum TallyPreconditionClauseResult {
+    MajorityOfValues(Vec<RadonTypes>, Vec<bool>),
+    MajorityOfErrors(RadonTypes),
```

Comment:
Why is this a `RadonTypes`, shouldn't it be a `RadonError`?

Reply:
Just because `mode` outputs `RadonTypes` and the result of this function is likely used as the result of the tally, which is also `RadonTypes`.